name_data/labeled/person_labeled.xml

```diff
@@ -3995,4 +3995,93 @@
+  <Name><PrefixMarital>Mr</PrefixMarital> <GivenName>JW</GivenName> <MiddleInitial>T</MiddleInitial> <Surname>BYRD</Surname></Name>
+  <Name><PrefixOther>Dr</PrefixOther> <GivenName>OLAMIDE</GivenName> <Surname>AKHUMERE</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>RIZA</GivenName> <MiddleName>TADDY</MiddleName> <Surname>CONROY</Surname></Name>
+  <Name><GivenName>VIVIANE</GivenName> <MiddleName>CHIOMA</MiddleName> <Surname>UKWU</Surname></Name>
+  <Name><PrefixOther>Dr</PrefixOther> <And>&amp;</And> <PrefixMarital>Mrs</PrefixMarital> <GivenName>ERIC</GivenName> <MiddleName>JOHN</MiddleName> <Surname>TRUEBLOOD</Surname> <SuffixOther>MD</SuffixOther></Name>
+  <Name><PrefixOther>Dr.</PrefixOther> <GivenName>SAFET</GivenName> <Surname>HATIC</Surname> <SuffixGenerational>II</SuffixGenerational></Name>
+  <Name><GivenName>TENZIN</GivenName> <Surname>DAWA</Surname></Name>
+  <Name><PrefixMarital>Ms</PrefixMarital> <GivenName>FRAVINIA</GivenName> <MiddleInitial>G</MiddleInitial> <Surname>AQUIPEL</Surname></Name>
+  <Name><GivenName>MEHRZAD</GivenName> <Surname>SERAJI</Surname></Name>
+  <Name><PrefixMarital>Ms.</PrefixMarital> <GivenName>SINDHU</GivenName> <MiddleName>BASAVAIAH</MiddleName> <Surname>IGALA</Surname></Name>
+  <Name><GivenName>FRANCIS</GivenName> <Surname>SCHWIEP</Surname></Name>
+  <Name><PrefixOther>Dr.</PrefixOther> <GivenName>IMEE</GivenName> <MiddleInitial>I</MiddleInitial> <Surname>IKALINA-PARAS</Surname></Name>
+  <Name><PrefixOther>Dr.</PrefixOther> <And>&amp;</And> <PrefixMarital>Mrs.</PrefixMarital> <GivenName>RHYS</GivenName> <MiddleInitial>P</MiddleInitial> <Surname>STRASIA</Surname></Name>
+  <Name><PrefixMarital>Ms.</PrefixMarital> <GivenName>ASMEETA</GivenName> <Surname>ERICKSON</Surname></Name>
+  <Name><PrefixOther>Dr.</PrefixOther> <GivenName>THRESSIAMMA</GivenName> <Surname>CHOLANKERIL</Surname> <SuffixOther>M.D.</SuffixOther></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>LEESA</GivenName> <MiddleName>ANN</MiddleName> <Surname>VITEK</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>EILEEN</GivenName> <Surname>JANEC,</Surname> <SuffixOther>MD</SuffixOther></Name>
+  <Name><PrefixMarital>Mr.</PrefixMarital> <And>&amp;</And> <PrefixMarital>Mrs.</PrefixMarital> <GivenName>EN</GivenName> <GivenName>SU</GivenName> <Surname>LIN</Surname> <SuffixOther>M.D.</SuffixOther></Name>
+  <Name><PrefixOther>Dr</PrefixOther> <And>&amp;</And> <PrefixMarital>Mrs</PrefixMarital> <GivenName>DERIC</GivenName> <MiddleInitial>C</MiddleInitial> <Surname>MCINTOSH</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>KELLY</GivenName> <MiddleName>MARIE</MiddleName> <Surname>JENSEN</Surname></Name>
+  <Name><GivenName>TOAN</GivenName> <MiddleName>QUOC</MiddleName> <Surname>TRAN</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>KATHLEEN</GivenName> <Surname>KIRCHMEIER</Surname> <Surname>KROESSLER</Surname></Name>
```

Comment:
Should `KIRCHMEIER` be a `MiddleName` in this case?

Reply:
According to the original CMS Physicians Data set for this individual, her middle_name is blank and last_name="KIRCHMEIER KROESSLER".

Also according to a glance at Google search results, KIRCHMEIER seems like a last name (like Smith, unlike Kathleen).